--- aces_container-1.0/CMakeLists.txt.orig	2013-07-27 01:07:55.000000000 +0200
+++ aces_container-1.0/CMakeLists.txt	2014-04-05 21:12:50.040574218 +0200
@@ -104,7 +104,7 @@
 find_package( PkgConfig )
 if ( PKG_CONFIG_FOUND )
 configure_file(config/AcesContainer.pc.in "${PROJECT_BINARY_DIR}/AcesContainer.pc" @ONLY)
-install( FILES "${PROJECT_BINARY_DIR}/AcesContainer.pc" DESTINATION lib/pkgconfig COMPONENT dev )
+install( FILES "${PROJECT_BINARY_DIR}/AcesContainer.pc" DESTINATION lib${LIB_SUFFIX}/pkgconfig COMPONENT dev )
 endif()
 
 include_directories(
